README.md

```diff
@@ -1,4 +1,3 @@
-
 ![postman icon](https://raw.githubusercontent.com/postmanlabs/postmanlabs.github.io/develop/global-artefacts/postman-logo%2Btext-320x132.png) 
```

Comment:
Add the new logo here instead of this old one.

Reply:
Add this text `Manage all of your organization's APIs in Postman, with the industry's most complete API development environment.`